sway-ir/src/verify.rs

```diff
@@ -292,13 +296,13 @@ impl<'a> InstructionVerifier<'a> {
         Ok(())
     }
 
-    fn verify_br(&self, dest_block: &Block) -> Result<(), IrError> {
-        if !self.cur_function.blocks.contains(dest_block) {
+    fn verify_br(&self, dest_block: &(Block, Vec<Value>)) -> Result<(), IrError> {
+        if !self.cur_function.blocks.contains(&dest_block.0) {
             Err(IrError::VerifyBranchToMissingBlock(
-                self.context.blocks[dest_block.0].label.clone(),
+                self.context.blocks[dest_block.0 .0].label.clone(),
```

Comment:
Here's an example too.  `self.context.blocks[dest_branch.block.0].label.clone()` would be more readable.